app/models/version.rb

```diff
@@ -243,14 +243,6 @@ def downloads_count
     gem_download.try(:count) || 0
   end
 
-  def runtime_dependencies_count
```

Comment:
Side note, does `length` always trigger another (cached) query on a loaded AR association? I didn't know this/was surprised.